patterns/1-initial/introducing-metrics-in-innersource.md

```diff
@@ -64,7 +64,18 @@ Produce a characterization of metrics as this might be helpful for others to und
 * Another class of metrics is around how much InnerSourcing is happening. A raw definition of InnerSource based on code submissions to a repository not owned by the submitter's team could be measurable in a few ways.
 * A third class of metrics are KPIs that we believe will improve the raw amount of InnerSourcing happening (e.g. mean-time-to-review, automated test runs on PRs, etc.).
 
-Note: The aforementioned metrics are some common examples. They aren't a one-size-fits-all. We expect the business goals of the organization to drive the metric selection process.
+Note: The aforementioned metrics are some common examples. They aren't a one-size-fits-all. We expect the business goals of the organization to drive the metric selection process. Below is the list consolidating some metrics candidates by business goals.
+| Business Goals | Metric Candidates |
+|---|---|
+| Remove silos and bottlenecks | # of PRs from external contributors<br># of PRs from external contributors that are merged<br># of outside collaborators<br># of issues created from external contributors<br># of reviews from external contributors |
+| Knowledge sharing | # of clone or forks from external guests<br># of visits from external guests (proposed by Jack) |
```

Comment:
Do you want to keep this "(proposed by Jack)" note in here?

We don't have to mention where which metric comes from. It is actually quite likely that many organizations have discover the same metrics over and over again anyways :)

Reply:
Sorry, that's typo. I should remove it from the sentence

Reply:
BTW, I will contact Wilson and Aaron to check if they concern on being mentioned in the page; Thanks for @spier 's review and reminding

Reply:
Confirmed no concern from Aaron, and waiting for Wilson's confirmation

Reply:
Update, just got reply from Wilson. Confirmed he has no concern on this.